tests/Museum/MuseumTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+use DinoApp\Museum\Museum;
+use DinoApp\Dinosaur\Dinosaur;
+use PHPUnit\Framework\TestCase;
+
+class MuseumTest extends TestCase
+{
+    public function testSuccessDisplayAllDinos()
+    {
+        $url = 'https://upload.wikimedia.org/wikipedia/commons/thumb/1/16/Maiasaura_skeleton.jpg/220px-Maiasaura_skeleton.jpg';
+        $testDino = new Dinosaur(1, 'test', 'Herbivore', 52.5, 420, 26.2, 9, 2,69, $url, 'herbivore.png');
```

Comment:
This should probably make use of mocking instead of using a real dinosaur! Meaning that you're only testing the Museum and not the dinosaur as well